src/Sylius/Behat/Context/Setup/ChannelContext.php

```diff
@@ -89,13 +99,34 @@ public function storeOperatesOnASingleChannel()
 
     /**
      * @Given /^the store operates on (?:a|another) channel named "([^"]+)"$/
+     * @Given the store operates on a channel identified by :code code
      */
-    public function theStoreOperatesOnAChannelNamed($channelName)
+    public function theStoreOperatesOnAChannelNamed($channelIdentifier)
     {
-        $channel = $this->channelFactory->createNamed($channelName);
-        $channel->setCode($channelName);
+        $channel = $this->channelFactory->createNamed($channelIdentifier);
+        $channel->setCode($channelIdentifier);
 
         $this->channelRepository->add($channel);
         $this->sharedStorage->set('channel', $channel);
     }
+
+    /**
+     * @Given the channel :channel is enabled
+     */
+    public function theChannelIsEnabled(ChannelInterface $channel)
+    {
+        $channel->enable();
+        $this->channelManager->flush();
+        $this->sharedStorage->set('channel', $channel);
+    }
+
+    /**
+     * @Given the channel :channel is disabled
```

Comment:
What about giving there parameter ``(enabled|disabled)`` and use ``$channel->setEnabled(true|false)``? It would allows us to do this two steps in one function ;)